### DISCUSSION TITLE
How can I get all the name of topics all repertorys in the github?

### DISCUSSION THREAD

**Discussion by zhuozhuoH** [community/community#22473] [New to GitHub]
<p>How can I get all the name of topics all  repertorys in the github?</p>

**Comment by jfullstackdev** [comment]
<p>you mean this ?</p>
<p><a href="https://github.com/topics">https://github.com/topics</a></p>

**Comment by zhuozhuoH** [comment]
<p>I mean I want to know how to get all topics list from github<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/b/9b4995fc3d49270be59b6e53c5424814d4429633.png" data-download-href="/uploads/short-url/m9JFe1YrZ5cdeP6vUdadLMImkYb.png?dl=1" title="屏幕截图 2022-03-06 162438" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75775454/181071300-89e6a0f2-247d-4b26-9474-dea23f582f2a.png" alt="屏幕截图 2022-03-06 162438" data-base62-sha1="m9JFe1YrZ5cdeP6vUdadLMImkYb" width="690" height="218" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9b4995fc3d49270be59b6e53c5424814d4429633_2_690x218.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9b4995fc3d49270be59b6e53c5424814d4429633_2_1035x327.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9b4995fc3d49270be59b6e53c5424814d4429633_2_1380x436.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9b4995fc3d49270be59b6e53c5424814d4429633_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">屏幕截图 2022-03-06 162438</span><span class="informations">1727×548 55.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>nope, those can be created by admins of a repo, it’s just a way to organize things</p>
<p>for example, if you have 10 repos that are Java development, you can create</p>
<p><code>zhuo02-java-dev</code></p>
<p>so that when you click that, it will collect all those 10 repos, if you used an existing topic, that will be part of that collection</p>

**Comment by zhuozhuoH** [comment]
<p>like the list<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/6/b62cd3f2e364efa51a952a50214c2f4595f37a77.png" data-download-href="/uploads/short-url/pZAWsVceCX7iaxHQSb9YankJWgT.png?dl=1" title="屏幕截图 2022-03-06 180823" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75775454/181071304-07903bcb-443a-4090-a64f-1bc4922090f2.png" alt="屏幕截图 2022-03-06 180823" data-base62-sha1="pZAWsVceCX7iaxHQSb9YankJWgT" width="690" height="402" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b62cd3f2e364efa51a952a50214c2f4595f37a77_2_690x402.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b62cd3f2e364efa51a952a50214c2f4595f37a77_2_1035x603.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b62cd3f2e364efa51a952a50214c2f4595f37a77_2_1380x804.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/6/b62cd3f2e364efa51a952a50214c2f4595f37a77_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">屏幕截图 2022-03-06 180823</span><span class="informations">1388×810 54.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
I want all topics list instead of only the most used

**Comment by jfullstackdev** [comment, marked as answer]
<p>yup, others are not yet curated, you may want to check this out,</p>
<p><a href="https://github.com/github/explore/blob/33d4d373d5194a534ff715edbc59ca8a696bca5f/CONTRIBUTING.md">https://github.com/github/explore/blob/33d4d373d5194a534ff715edbc59ca8a696bca5f/CONTRIBUTING.md</a></p>
<p>and this</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/explore">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/explore" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181071306-af57863f-a405-4aa8-ac12-ee1aca5c3d29.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/f/5feca4309766c265e03790f5611330443b2e9368_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/f/5feca4309766c265e03790f5611330443b2e9368_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/f/5feca4309766c265e03790f5611330443b2e9368.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/f/5feca4309766c265e03790f5611330443b2e9368_2_10x10.png"></div>

<h3><a href="https://github.com/github/explore" target="_blank" rel="noopener">GitHub - github/explore: Community-curated topic and collection pages on GitHub</a></h3>

  <p>Community-curated topic and collection pages on GitHub - GitHub - github/explore: Community-curated topic and collection pages on GitHub</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by zhuozhuoH** [comment]
<p>OK,thank you very much!</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="zhuo02" data-post="5" data-topic="234333">
<div class="title">
<div class="quote-controls"></div>
 zhuo02:</div>
<blockquote>
<p>I want all topics list instead of only the most used</p>
</blockquote>
</aside>
<p>I already sent the link, I think that’s the popular Topics on GitHub</p>
<p><a href="https://github.com/topics">https://github.com/topics</a></p>

**Comment by zhuozhuoH** [comment]
<p>OK,I will see.Thanks!!!</p>
